python/tests/ibd.py

```diff
@@ -126,22 +126,37 @@ def add(self, other):
 class IbdFinder:
     """
     Finds all IBD relationships between specified samples in a tree sequence.
+    One of either sample_pairs or samples must be provided
     """
 
-    def __init__(self, ts, samples=None, min_length=0, max_time=None):
+    def __init__(
+        self, ts, sample_pairs=None, samples=None, min_length=0, max_time=None
```

Comment:
Let's keep it simple for now and just accept ``sample_pairs``. We can add the all-pairs behaviour later at the top level.

Reply:
Okay, cool. This addresses one of the points I made below.